### DISCUSSION TITLE
Blue dot on the Code button

### DISCUSSION THREAD

**Discussion by carlomilanesi** [community/community#42081] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

What is the meaning of the blue dot appearing in the top right vertex of the "Code" button in the main page of a repository?

**Comment by arobert-da** [comment]
According to [a Reddit thread](https://www.reddit.com/r/github/comments/ytth8a/whats_this_blue_dot/), it is a "new" indicator to inform you about Codespaces. Once this is selected under the dropdown and the page reloaded, the blue dot is gone!
